e2e/cypress/integration/enterprise/cloud/Billing/subscriptions_spec.js

```diff
@@ -0,0 +1,250 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Please user lower case in file name.

Reply:
renamed Billing folder to billing folder